### PR TITLE
Feature/jjdd8 fpo 13mb

### PR DESCRIPTION
**Pull request by michal-brakon**

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/AppPropertiesReader.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    private Properties properties = new Properties();
+
+    public String getProp(String config) {
+        try {
+            File file = new File("config.properties");
+            FileInputStream fileInput = new FileInputStream(file);
+            properties.load(fileInput);
+            fileInput.close();
+        } catch (FileNotFoundException e) {
+            System.out.println("Brak pliku!");
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/AppPropertiesReader.java]: sout? ;)

File: src/main/java/com/infoshareacademy/AppPropertiesReader.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        } catch (IOException ex) {
+            System.out.println("Blad odczytu pliku!");
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/AppPropertiesReader.java]: sout? ;)

File: src/main/java/com/infoshareacademy/BookSorter.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BookSorter {
+    private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
+
+    private AppPropertiesReader propertiesReader = new AppPropertiesReader();
+
+    public List<Book> sortingByTitle(List<Book> books) {
+        if (propertiesReader.getProp("sortByTitle").equals("ASC")) {
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/BookSorter.java]: wyrzućmy tym sortowania ASC / DESC do enuma

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: src/main/java/com/infoshareacademy/AppPropertiesReader.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class AppPropertiesReader {
+    private static final Logger stdout = LoggerFactory.getLogger("CONSOLE_OUT");
+    private Properties properties = new Properties();
+
+    private File file = new File("config.properties");
```
Comment (rafalmisiak) [inline src/main/java/com/infoshareacademy/AppPropertiesReader.java]: nazwę pliku wynieś proszę do stale

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**